src/NzbDrone.Core/Notifications/DownloadMessage.cs

```diff
@@ -8,6 +8,7 @@ public class DownloadMessage
     {
         public string Message { get; set; }
         public Series Series { get; set; }
+        public Episode Episode { get; set; }
```

Comment:
Looks like this isn't used, nor is it valid since there could be more than one episode in a file. `EpisodeFile.Episodes` should be used instead.

Reply:
yes, i did use episodeFile.Episodes in the end. I am guessing I just added this line in by accident and then didn't remove it - and it didn't give any build errors so I didn't notice. I'll remove it.